### PR TITLE
Redux percentage tool for converting to and from needed values

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

_Provide a description of what this PR does. Link to any relevant GitHub issues, Notion tasks or Discord discussions._

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2156** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2156" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2156?utm_source=stack-comment).

File: src/components/Create/components/pages/ProjectToken/hooks/ProjectTokenForm.ts
Comment (tomquirk) [inline src/components/Create/components/pages/ProjectToken/hooks/ProjectTokenForm.ts:49]: Wanna make mention of this just in case: if we're gonna store percentages in redux, gotta make sure that the data is transposed to the correct units before passing to the `launch` transaction. Using functions like `parseReservedRate` etc.

**Review by tomquirk — APPROVED** [review]

**Review by wraeth-eth — COMMENTED** [review]
@tomquirk thanks for the tip! I'll add this is a focus exercise in my todo list of related work for this!

**Comment by wraeth-eth** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2156?utm_source=stack-comment).


Job Reference: vtVBnYDD1aIW70DZJ1cT

**Pull request merged**
